coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -75,9 +87,54 @@ def get_form_kwargs(self):
         kwargs["initial"] = form_data
         return kwargs
 
-    def form_valid(
-        self, form: UpdateAllocationForm, parent_allocation: Optional[Allocation] = None
-    ):
+    def _acl_reset_message(self):
+        name  = Allocation \
+                    .objects \
+                    .get(pk=self.kwargs.get("allocation_id")) \
+                    .get_attribute(name='storage_name')
+        return (
+            f'ACL reset initiated for {name}.  '
+            'An e-mail notification will be sent when the task completes.'
+        )
+
+    def _allocation_linkage_exists(self):
+        has_linkage = True
```

Comment:
@ris-bmulligan would you consider getting a result set and check if it is empty? This way the try-except would not be necessary since all it's required is to determined whether or not some records exists.